src/main/java/appeng/core/settings/TickRates.java

```diff
@@ -45,6 +49,8 @@
 
 	StorageBus( 5, 60 ),
 
+	FluidStorageBus( 5, 60 ),
+
 	ItemTunnel( 5, 60 ),
 
 	LightTunnel( 5, 60 ),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)